website/src/menu.tsx

```diff
@@ -16,20 +16,28 @@ import {
   drawerItem,
   drawerList,
   navButton,
+  navButtonChoices,
   navDrawer,
   navLink,
   navMenu,
+  subMenuItems,
 } from "./menu.css"
 import { closeBlock } from "./sprinkles.css"
 
-export const NavMenu = () => {
+export const NavMenu = (p: { menuID: number }) => {
   const location = useLocation()
-  const [{ data }] = Wordpress.useMainMenuQuery()
-  const menuItems = data?.menuItems?.nodes
+  const [{ data }] = Wordpress.useMenuByIdQuery({
+    variables: { slug: p.menuID },
```

Comment:
I think we can call the query variable `id` instead of `slug` because its not a slug right?